### ISSUE TITLE
Bomn notation

### ISSUE THREAD

**Issue by Diaskhan** [danielgerlag/workflow-core#85]
It is posibble to make bpmn solution with this workflow?
Human tasks, paralell task, condition execution ? 
Script tasks, web service tasks ?

**Comment by danielgerlag** [comment]
Absolutely, the core library provides the underlying workflow infrastructure, and then you can build various components such as `Web service call`, `scripts`, etc...
